main/adapters/http_server/verification_service.go

```diff
@@ -1,24 +1,68 @@
 package http_server
 
 import (
+	"context"
 	"net/http"
 
 	"github.com/google/uuid"
+
+	log "github.com/sirupsen/logrus"
 )
 
+type Verify func(ctx context.Context, hash []byte) HTTPResponse
+type VerifyOffline func(upp []byte, hash []byte) HTTPResponse
+
 type VerificationService struct {
-	Verify func([]byte) HTTPResponse
+	Verify
+	VerifyOffline
 }
 
-var _ Service = (*VerificationService)(nil)
+func (s *VerificationService) HandleRequest(offline, isHashRequest bool) http.HandlerFunc {
```

Comment:
Same remark as in https://github.com/ubirch/ubirch-client-go/pull/102/files#diff-440fd7ece871c65eb098415fc61091366e9bd354bed520ce3d08206850b7c2c0R30

Reply:
I tried: https://github.com/ubirch/ubirch-client-go/pull/102/commits/5f22fe05f638f482fbb668e3edab843e64915007